packages/enhanced/src/lib/container/runtime/FederationRuntimeEntryDependency.ts

```diff
@@ -0,0 +1,38 @@
+import { normalizeWebpackPath } from '@module-federation/sdk/normalize-webpack-path';
+
+const makeSerializable = require(
+  normalizeWebpackPath('webpack/lib/util/makeSerializable'),
+);
+const EntryDependency = require(
+  normalizeWebpackPath('webpack/lib/dependencies/EntryDependency'),
+) as typeof import('webpack/lib/dependencies/EntryDependency');
+
+class FederationRuntimeEntryDependency extends EntryDependency {
+  // eslint-disable-next-line no-useless-constructor
+  constructor() {
+    const code = `import * as runtime from '@vmok/runtime';
+		__webpack_require__.vmok.runtime = runtime;
```

Comment:
Do we still need `vmok` references here for backward compat? (its fine if we do but maybe place a comment somewhere to explain it